Add 25px and 50px Layout Builder section minimum-height options.

The min_height section option (bootstrap_styles.settings) offered 100px-800px.
CAS needs two shorter steps for tight spacers, so prepend osu-min-h-25 and
osu-min-h-50. The matching CSS ships in the manzanita theme (madrone's
$min-heights map only generates 100-800).
--- a/config/install/bootstrap_styles.settings.yml
+++ b/config/install/bootstrap_styles.settings.yml
@@ -46,4 +46,4 @@
 scroll_effects_data_key: data-aos
 scroll_effects_library_type: external
 items_alignment: "osu-align-items-stretch|Stretch\r\nosu-align-items-start|Top\r\nosu-align-items-center|Center\r\nosu-align-items-end|Bottom"
-min_height: "osu-min-h-100|100px\r\nosu-min-h-200|200px\r\nosu-min-h-300|300px\r\nosu-min-h-400|400px\r\nosu-min-h-500|500px\r\nosu-min-h-600|600px\r\nosu-min-h-700|700px\r\nosu-min-h-800|800px"
+min_height: "osu-min-h-25|25px\r\nosu-min-h-50|50px\r\nosu-min-h-100|100px\r\nosu-min-h-200|200px\r\nosu-min-h-300|300px\r\nosu-min-h-400|400px\r\nosu-min-h-500|500px\r\nosu-min-h-600|600px\r\nosu-min-h-700|700px\r\nosu-min-h-800|800px"
